src/compiler/utilities.ts

```diff
@@ -211,8 +211,9 @@ module ts {
             isCatchClauseVariableDeclaration(declaration);
     }
 
-    export function getEnclosingBlockScopeContainer(node: Node): Node {
-        let current = node;
+    // Gets the nearest enclosing block scope container that has the provided node as a descendant, that is not the provided node.

```

Comment:
While I agree with JSDoc, it doesn't look like we are using it anywhere else in the file.